lib/mix/tasks/chatkit_room.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Mix.Tasks.Cadet.ChatkitRoom do
+  @moduledoc """
+    Creates ChatKit rooms for answers with empty comments in the database.
+    Room creation: https://pusher.com/docs/chatkit/reference/api#create-a-room
+    Status codes: https://pusher.com/docs/chatkit/reference/api#response-and-error-codes
+
+    Note:
+    - Task is to run daily
+  """
+  use Mix.Task
+
+  import Mix.EctoSQL
+
+  alias Cadet.Repo
+  alias Cadet.Assessments.Submission
+  alias Cadet.Chat.Room
+
+  def run(_args) do
+    ensure_started(Repo, [])
+    HTTPoison.start()
+
+    Submission
+    |> Repo.all()
+    |> Enum.each(fn submission -> Room.create_rooms(submission) end)
```

Comment:
Filter the submissions at the database level. We shouldn't fetch the entire submission table every time.

```elixir
Submission
|> where(status: ^:submitted)
|> Repo.all()
|> Enum.each(fn submission -> Room.create_rooms(submission) end)
```

I think this will suffice, but there is definitely a better way to do this (filter submissions by whether the associated answers have comment)

Reply:
Can I check if graders are able to access submissions even if they are only attempted? Or are they only able to access the submissions that have already been submitted?

Reply:
At the moment, graders only see student's attempts to solve an assessment after the attempt is submitted. Just attempting and saving currently does not give the graders (or any staff) access to the program.

Reply:
Thank you for the clarification! Changes have been made accordingly.